Assignments/ManageClass/manageClass.txt

```diff
@@ -0,0 +1,32 @@
+<apex:page controller="manageClassController" lightningStylesheets="true">
```

Comment:
controller name should be camel case starting with capital letters

Reply:
Understood.